From: Changwoo Ryu <cwryu@debian.org>
Date: Sun, 3 Nov 2019 21:28:48 +0100
Subject: Do not use intersphinx

Forwarded: not-needed
Last-Update: 2019-07-15

According to the Debian policy 3.9.7, the build should not attempt network
connection.
---
 docs/conf.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index bebbf93..ccbe354 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -63,7 +63,7 @@ if os.environ.get('READTHEDOCS', 0):
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx',
+extensions = ['sphinx.ext.autodoc', #'sphinx.ext.intersphinx',
               'sphinx.ext.extlinks', 'sphinx.ext.todo']
 
 # Add any paths that contain templates here, relative to this directory.
@@ -260,7 +260,7 @@ man_pages = [
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+#intersphinx_mapping = {'http://docs.python.org/': None}
 
 extlinks = {
     'issue': ('https://github.com/emcconville/wand/issues/%s', '#'),
